src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -6,6 +6,16 @@
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null || user.getId() == null
+                || user.getLogin() == null || user.getAge() == null
+                || user.getPassword() == null || user.getAge() < 18
+                || user.getAge() <= 0 || user.getLogin().length() < 6
+                || user.getPassword().length() < 6 || user.getLogin().contains(user.getLogin())
```

Comment:
`user.getLogin().contains(user.getLogin())` this check does not make sense, we need to check if there is a user with this login in the storage. We need to use method `User get (String login)` of StorageDaoImpl class for this purpose